sdks/python/apache_beam/coders/row_coder.py

```diff
@@ -182,7 +188,6 @@ def encode_to_stream(self, value, out, nested):
           words[i // 8] |= is_null << (i % 8)
 
     self.NULL_MARKER_CODER.encode_to_stream(words.tobytes(), out, True)
-
     for c, field, attr in zip(self.components, self.schema.fields, attrs):
```

Comment:
Don't we also need to adjust the order for `self.schema.fields` and `attrs` to be consistent with the order of `self.components`? Maybe I'm missing something though, since this is passing tests.

Reply:
Yes, I added to.